src/formatter.js

```diff
@@ -0,0 +1,70 @@
+const MESSAGE = Symbol.for('message');
+// eslint-disable-next-line import/no-extraneous-dependencies
+const yamlifyObject = require('yamlify-object');
+// eslint-disable-next-line import/no-extraneous-dependencies
+const yamlifyColors = require('yamlify-object-colors');
+
+const utils = require('./utils');
+const Message = require('./message');
+const Colorizer = require('./colorizer');
+const defaultErrorColors = require('./colors');
+
+const { format } = require('winston');
+
+module.exports = function WinstonConsoleFormatter(options = {}) {
+  options = Object.assign({}, options, {
+    prefix: '',
+    postfix: '',
+    stackTrace: true,
+    colors: defaultErrorColors,
+    types: yamlifyColors,
+  });
+  return format(info => {
+    var remainingInfo = Object.assign({}, info);
+    delete remainingInfo.from;
+    delete remainingInfo.stack;
+    delete remainingInfo.trace;
+    delete remainingInfo.message;
+    delete remainingInfo.timestamp;
+    delete remainingInfo.label;
+    delete remainingInfo.level;
```

Comment:
It looks like this project uses the spread syntax, so this file can too. e.g. this can be expressed as:

```js
module.exports = function ConsoleFormatter({
  postfix = '',
  prefix = '',
  stackTrace = true,
  colors = defaultErrorColors,
  types = yamlifyColors,
}) {
  return format(info => {
    const { level, label, message, timestamp, from, stack, trace, ...remainingInfo } = info;
    //...
  })
};

Reply:
I originally had it that way, but something broke, I feel like tests or something, I'll certainly try it again.

Reply:
```  logger3 ●  …/git/halkeye/winston-console-formatter  npm run test

> winston-console-formatter@1.0.0-beta.3 test /home/halkeye/go/src/github.com/halkeye/winston-console-formatter
> lab -T node_modules/babel-lab test/suites/*.js -l

/home/halkeye/go/src/github.com/halkeye/winston-console-formatter/node_modules/babel-core/lib/transformation/file/index.js:590
      throw err;
      ^

SyntaxError: unknown: Unexpected token (24:66)
  22 |     }, options);
  23 |
> 24 |     const { level, label, message, timestamp, from, stack, trace, ...remainingInfo } = info;
     |                                                                   ^


```

Yea, the tests don't run though babel, so it doesn't transpile them, and I think Object explode is only in 10

Reply:
Oh well, someone can solve that in the future of they want to :)